sycl/source/backend/opencl.cpp

```diff
@@ -44,6 +46,66 @@ __SYCL_EXPORT queue make_queue(const context &Context,
   return detail::make_queue(NativeHandle, Context, nullptr, false,
                             ContextImpl->get_async_handler(), backend::opencl);
 }
+
+// Free functions to query OpenCL backend extensions
+// TODO: Extensions have been deprecated for aspects
```

Comment:
Could you please clarify what this `TODO` means, i.e. what is the action we supposed to take in the future? My understanding is that even though device extensions are deprecated in SYCL in favor of device aspects, OpenCL extensions are not going anywhere and for that we have this backend-specific interface.